### ISSUE TITLE
Arbiter2 ignoring whitelist

### ISSUE THREAD

**Issue by kdalenberg** [CHPC-UofU/arbiter2#7]
I have arbiter2 running on some Centos 7 systems in debug mode, configured to use the whitelist file that comes with the install.  I am getting alerts that a user is being flagged for rclone, but that process is in the whitelist file.

How can I troubleshoot why the whitelist file is being ignored?  I did add some entries to that file with a comment, are comments allowed in that file?

**Comment by jay-mckay** [comment]
@kdalenberg Thanks for bringing this to our attention.

I believe all lines in that file are treated directly as process names, meaning no comments (this is an oversight on our part). 

As far as troubleshooting, where do you have the `rclone` process defined as whitelisted? Is it in a file, or defined in a list in the config file? Additionally, under which heading is this defined, i.e. `[processes]` some status group like `[status.admin]`? 

If you run the following python script in the `arbiter` directory it will should show you what you think the whitelist is.

```
import cfgparser

cfgparser.load_config("../etc/config.toml") # path to your config file

import user # implicitly creates whitelist

for key, value in user.whitelist.items():
    print(f"{key}:{value}\n")
```

If the process is there and the user is still being flagged, let me know.

**Comment by kdalenberg** [comment]
@jay-mckay Just this morning I noticed in one of the admin emails for the "node" process that I had whitelisted that it had a single asterisk and that is tied to this message at the bottom of the email:

"*This process is generally permitted on interactive nodes and is only counted against you when considering memory usage (regardless of the process, too much memory usage is still considered bad; it cannot be throttled like CPU). The process is included in this report to show usage holistically."

After reading that, I observed that processes that still exceed the memory quota can get flagged.  Makes perfect sense.

I will remove the comment line from our whitelist.txt file, I would suggest putting a note in the documentation about not having comments in that file.
